js/components/quack.jsx

```diff
@@ -16,10 +14,6 @@ var Quack = React.createClass({
         <div className="channel-box">
           <ChannelMessages>
           </ChannelMessages>
-          <ChannelUsers>
-          </ChannelUsers>
-          <Input>
-          </Input>
```

Comment:
i was gonna say, on top of that, we lose the idea of a component that's ONLY responsible for the messages box.. it might make sense to have a Channel component, which contains ChannelMessages, Input and ChannelUsers